announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm

```diff
@@ -250,11 +250,15 @@
 						#foreach ($ann_item in $showMessagesList)
 							#set ($rowCount =$rowCount + 1)
 							#set($ann_item_props=$ann_item.getProperties())
+							#set($highlight = $ann_item_props.getProperty("highlight"))
 
 							<tr #rowTRconstruct($ann_item_props $displayOptions $ann_item.Header.draft) >
 
-								<th headers="subject" scope="row">
-									<strong>
+								#if ($highlight == "true")
```

Comment:
Can you explain this to me? Why are you setting th o td elements based on the hightlight value?

Reply:
It is because the th set bold text and the td set normal text. So i did that all elements use normal text. But the highlighted announcements use the bold text

Reply:
Don't change this, lets look for a solution via CSS, maybe create a new class and make the text a bit bigger, 1 or 2 px.